tests/ttnn/unit_tests/operations/test_moreh_softmax.py

```diff
@@ -346,8 +335,17 @@ def test_softmax_callback(shape_dim_strategy, device):
     dev_x = ttnn.Tensor(x, ttnn.bfloat16).to(ttnn.TILE_LAYOUT).to(device)
 
     tt_cpu = torch.softmax(x, dim)
+
     for i in range(2):
+        dev_x = ttnn.Tensor(x, ttnn.bfloat16).to(ttnn.TILE_LAYOUT).to(device)
         tt_npu = ttnn.operations.moreh.softmax(dev_x, dim, strategy=strategy)
+        if i == 0:
+            num_program_cache_entries = device.num_program_cache_entries()
+            assert num_program_cache_entries > 0
+        else:
+            assert device.num_program_cache_entries() == num_program_cache_entries
+        torch_dummy = torch.randn([32, 32])
```

Comment:
Please modify this as you did with the softmin forward callback.

Reply:
Resolved